nautobot/docs/user-guide/administration/upgrading/from-v1/region-and-site-data-migration-guide.md

```diff
@@ -1,3 +1,4 @@
+[//]: # (TODO&#40;timizuo&#41;: Move file to /docs/development/migration)
```

Comment:
Not the whole file - probably just everything under the `## Region and Site Related Data Model Migration Guide For Existing Nautobot App installations` heading and its descendants.